alembic/versions/58115a0c85ac_add_default_atat_role.py

```diff
@@ -0,0 +1,37 @@
+"""add default atat role
+
+Revision ID: 58115a0c85ac
+Revises: 428edc7b735c
+Create Date: 2018-07-19 17:13:32.881309
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm.session import Session
+
+from authz.models.role import Role
+from authz.models.permissions import Permissions
+
+# revision identifiers, used by Alembic.
+revision = '58115a0c85ac'
+down_revision = '428edc7b735c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    session = Session(bind=op.get_bind())
+    mission_owner_role = Role(
+        name='default',
```

Comment:
Can we name this `mission_owner` instead of `default`? I think the name `default` is a little too presumptive about the user model.

Reply:
We can, but I actually just changed it from `mission_owner` to `default`. After talking with Hardik I was worried that `mission_owner` might be confusing. Can you think of another name, or do you think it won't be an issue?

Reply:
After chatting about the name, I agree that "default" is good here. Sorry for the confusion!